Sample Code/Time of Flight/Readme.md

```diff
@@ -2,30 +2,11 @@
 
 This is a sample of JavaScript code used to retrieve information from the time of flight sensors via WebSockets. There are four time of flight sensors on Misty I: left, center, right, and back. The left, center, and right sensors are all located in the front. This sample code subscribes to the center time of flight sensor only.
 
-## Subscribing to a WebSocket Data Stream
+To run this sample:
 
-To subscribe to a WebSocket data stream, you must first open the WebSocket. Then, send a message to specify the exact data you want to receive. For some WebSocket data, you must also send a REST command to the robot to enable the systems that start generating that data. For the time-of-flight sensor data that the [`tofApp.js` sample](https://github.com/MistyCommunity/SampleCode/tree/master/Time%20of%20Flight) uses, sending a REST command is not required, because Misty's time-of-flight sensors are always on.
+1. Replace the `ip` variable in the `tofApp.js` file with your own robot's IP address.
+2. Open the `index.html` file in your web browser.
 
-The first thing the `tofApp.js` sample does is to construct the message that subscribes to the exact WebSocket data we want.
+You can view messages from this app by opening the developer console in your web browser window.
 
-The `Type` property is the name of the desired event type (or data stream) to receive messages from. Misty's available event types are described in detail in the [Event Types](../../../misty-ii/reference/sensor-data) documentation. You can subscribe to each of these event types via a WebSocket connection.
-
-The optional `DebounceMs` value specifies how frequently the data is sent. For time-of-flight data, if you don't specify a value, by default the data is sent every 250ms. In this case, we've set it to be sent every 100ms.
```

Comment:
I just ran a quick test and it looks like each edge TOF messages are generated at 6.6 Hz and each obstacle TOF message at 1.6 Hz (833 ms). And independent of that, setting a debounce that is less than the 'default' is kind of meaningless. The debounce can slow things down, but it can't speed things up.

Reply:
Cool -- I didn't know that about the ToFs. Fixing copy.

Reply:
Actually, I think I will leave that out of this README, and make sure the information we have in the docs is accurate, instead.